ql/src/java/org/apache/hadoop/hive/ql/parse/ReplicationSemanticAnalyzer.java

```diff
@@ -151,21 +154,49 @@ private void setReplDumpTablesList(Tree replTablesNode) throws HiveException {
       }
 
       if (listIdx == 0) {
-        LOG.info("ReplScope: Set Included Tables List: {}", tablesList);
+        LOG.info("{} ReplScope: Set Included Tables List: {}", replScopeType, tablesList);
         replScope.setIncludedTablePatterns(tablesList);
       } else {
-        LOG.info("ReplScope: Set Excluded Tables List: {}", tablesList);
+        LOG.info("{} ReplScope: Set Excluded Tables List: {}", replScopeType, tablesList);
         replScope.setExcludedTablePatterns(tablesList);
       }
     }
   }
 
+  private void setOldReplPolicy(Tree oldReplPolicyTree) throws HiveException {
+    oldReplScope = new ReplScope();
+    int childCount = oldReplPolicyTree.getChildCount();
+
+    // First child is DB name and optional second child is tables list.
+    assert(childCount <= 2);
+
+    // First child is always the DB name. So set it.
+    oldReplScope.setDbName(oldReplPolicyTree.getChild(0).getText());
+    LOG.info("Old ReplScope: Set DB Name: {}", oldReplScope.getDbName());
+    if (!oldReplScope.getDbName().equalsIgnoreCase(replScope.getDbName())) {
+      LOG.error("DB name {} cannot be replaced to {} in the replication policy.",
```

Comment:
As discussed yesterday, we can keep dbName for readability. Also, it will be helpful to keep it complete repl policy if in future we want to support it for some use case.